distributed/dashboard/tests/test_scheduler_bokeh.py

```diff
@@ -54,22 +55,8 @@ def test_simple(c, s, a, b):
     yield gen.sleep(0.1)
 
     http_client = AsyncHTTPClient()
-    for suffix in [
-        "system",
-        "counters",
-        "workers",
-        "status",
-        "tasks",
-        "stealing",
-        "graph",
-        "individual-task-stream",
-        "individual-progress",
-        "individual-graph",
-        "individual-nbytes",
-        "individual-nprocessing",
-        "individual-profile",
-    ]:
-        response = yield http_client.fetch("http://localhost:%d/%s" % (port, suffix))
+    for suffix in applications:
+        response = yield http_client.fetch("http://localhost:%d%s" % (port, suffix))
```

Comment:
This can be primarily a matter of personal preference, but wouldn't it make more sense to keep the format `%d/%s` and remove the `/` prefix from the strings in `applications`? My reasoning for that is that we can never append a string to the port without the `/`, so I think it makes this more error prone (if someone adds a new suffix and forgets the `/`).

Reply:
Perhaps, but I wasn't strongly confident about why the leading slash was present in applications to begin with, and so I decided that that choice should be made in another PR.  

Paths in web servers are sometimes odd, especially when using prefixes.  I'd rather just make the one change to avoid confusion.

Reply:
Sounds good, thanks for the info.